torch/utils/_config_module.py

```diff
@@ -47,6 +75,12 @@ def visit(
                 config[name] = _ConfigEntry(default=value)
                 if dest is module:
                     delattr(module, key)
+            elif isinstance(value, Config):
+                config[name] = _ConfigEntry(
+                    default=value.default, justknob=value.justknob
```

Comment:
we're probably going to add a lot of these soon, better to feed it straight to `_ConfigEntry`

Reply:
Sorry - Do you mean make a __init__ on _ConfigEntry that eats a config? If so done.

If not, let me know what you want here :).

Reply:
yes